src/components/Shared/RevisionSearch.tsx

```diff
@@ -17,23 +18,23 @@ import SearchResultsList from '../Search/SearchResultsList';
 
 function RevisionSearch(props: RevisionSearchProps) {
   const [focused, setFocused] = useState(false);
-  const { searchResults, view } = props;
+  const { prevRevision, searchResults, setPopoverIsOpen, view } = props;
+
   const dispatch = useAppDispatch();
+  const { replaceSelectedRevision } = useSelectRevision();
 
   const handleFocus = (e: MouseEvent) => {
     if (
       (e.target as HTMLElement).matches(
-        `#search-revision-input, 
-          #search-results-list, 
-          #search-results-list *`,
+        `#revision-search-container, 
+        #revision-search-container *`,
+      ) &&
+      // do not open search results when cancel button is clicked
+      !(e.target as HTMLElement).matches(
```

Comment:
[question] are we 100% sure all users will use firefox for this tool? Does Safari/Chrome have the same API?